src/Directory.Build.props

```diff
@@ -5,7 +5,7 @@
     <Copyright>Sebastien Ros</Copyright>
     <Authors>Sebastien Ros</Authors>
 
-    <LangVersion>latest</LangVersion>
+    <LangVersion>11</LangVersion>
```

Comment:
maybe could add comment why needed so can be later `latest` when the needed feature is in latest default SDK

Reply:
As a matter of fact, this change is not necessary, I just like to be explicit about the C# version required for building the project. To my understanding, the meaning of `latest` depends on the .NET SDK installed, which just doesn't seem like a good idea because one may have an older SDK installed, which resolves `latest` to an older C# version than required. (E.g. the highest target we have is .NET 6 but that only includes C# 10.) Please correct me if I'm wrong or if you'd just like to have `latest` and I'll revert it.

Reply:
OK, maybe the explicit version will make things more.. explicit 👍🏻 I've just used to having latest SDKs and (hoping to have latest) CI tools.